tools/wpt/browser.py

```diff
@@ -151,9 +156,15 @@ def find_binary_path(self, path=None):
             "Darwin": "mac"
         }.get(uname[0])
 
+        application_name = {
```

Comment:
I feel this ought to be factored into a single place, but let's do that in a followup.